docs/docs/reference/docgen/javascript/auth/content/hasura-auth-client/content/11-on-auth-state-changed.mdx

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Please delete the auto-generated documentation. DocGen will run when building Docusaurus, so these will be generated on-the-fly.

Reply:
Ah, I was wondering why these files were here. Maybe we could have docgen delete existing autogenerated files when generating new ones.

Will fix.

Reply:
We are not running it in a pre-commit hook anymore, so once these are removed, we won't see them again being tracked. I think these are here because you made some changes that resulted in a few auto-generated documentation files, but in the meantime, Pilou disabled DocGen in the pre-commit hook and deleted all the existing files to improve the review experience.